src/app/speechtotext/speechtotext.component.spec.ts

```diff
@@ -1,14 +1,30 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-
 import { SpeechtotextComponent } from './speechtotext.component';
 import { SpeechService } from "../services/speech.service";
-import { StoreModule } from "@ngrx/store";
+import { StoreModule, Store } from "@ngrx/store";
 import { reducer } from "../reducers/index";
+import * as fromRoot from '../reducers';
 import { RouterTestingModule } from "@angular/router/testing";
+import { Router } from '@angular/router';
+import * as queryactions from '../actions/query';
+import * as speechactions from '../actions/speech';
 
 describe('SpeechtotextComponent', () => {
   let component: SpeechtotextComponent;
   let fixture: ComponentFixture<SpeechtotextComponent>;
+  let message: any;
+  let ticks: any;
+  let timer: any;
```

Comment:
Don't just define type as ```any```, always try to declare exact type of variable. And please also fix it at all instances.

Reply:
any is used in component.ts file i m just keeping it sync with it
`